Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/common/certificate/CertificatePersonIdentifier.kt

```diff
@@ -8,28 +9,28 @@ import org.joda.time.LocalDate
 import timber.log.Timber
 
 data class CertificatePersonIdentifier(
-    val dateOfBirth: LocalDate,
-    val lastNameStandardized: String,
-    val firstNameStandardized: String?
+    @SerializedName("dateOfBirth") val dateOfBirth: LocalDate,
+    @SerializedName("familyNameStandardized") val lastNameStandardized: String,
+    @SerializedName("givenNameStandardized") val firstNameStandardized: String?
 ) {
 
     /**
      * Used internally to idenitify and store the data related to this person.
      */
-    internal val code: String by lazy {
-        val dob = dateOfBirth.toString()
-        val lastName = lastNameStandardized
-        val firstName = firstNameStandardized
-        "$dob#$lastName#$firstName"
-    }
```

Comment:
`lazy` does not play nice with `GSON`, on deserilzation it is null. We can still cache it by using a pattern like here:

https://github.com/corona-warn-app/cwa-app-android/blob/64fa93a4bd99b2084fb1e876c9532d5182362881/Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/common/certificate/DccV1.kt#L39-L42

but I'm not sure if it's worth it in this case?